interpreter/cling/lib/Interpreter/ValuePrinter.cpp

```diff
@@ -369,42 +369,52 @@ static std::string printUnpackedClingValue(const Value& V) {
 namespace cling {
 
   std::string printValue(void* ptr) {
-    std::ostringstream strm;
     if (!ptr) {
-      strm << "<<<NULL>>>";
+      return "nullptr";
     } else {
+      std::ostringstream strm;
       strm << ptr;
-      if (isAddressValid(ptr))
-        strm << " VALID";
-      else
-        strm << " INVALID";
+      if (!isAddressValid(ptr))
+        strm << " <invalid memory address>";
+      return strm.str();
     }
-    return strm.str();
   }
 
   // Bool
   std::string printValue(bool val) {
-    std::ostringstream strm;
-    strm << std::boolalpha;
-    strm << val;
-    strm << std::noboolalpha;
-    return strm.str();
+    if (val) {
+      return "true";
+    } else {
+      return "false";
+    }
```

Comment:
Alternatively, `return val ? "true" : "false"` would have done :-) But details, whatever you prefer.